lib/fastlane/plugin/revenuecat_internal/helper/versioning_helper.rb

```diff
@@ -35,7 +35,7 @@ def self.determine_next_version_using_labels(repo_name, github_token, rate_limit
           end
         end
         UI.important("Type of bump after version #{old_version} is #{type_of_bump}")
-        increase_version(old_version, type_of_bump, false)
+        return increase_version(old_version, type_of_bump, false), type_of_bump
```

Comment:
Not sure which is better, if this or returning a hash

Reply:
Hmm honestly, I've never been much of a fan of returning multiple values for the same function... I would prefer to return an object... What do you think about separating the action into 2, one to get the type of bump from the changelog, and another to calculate the next version? It's true that it will slightly more logic to the sdks, but I feel it would be cleaner

Reply:
Do you mind if I go with the returning of an object for now?

Reply:
Ok, I think I can just make this function return the type of change and expose increase version as an action

Reply:
👍 Thanks! I'm ok merging this if we want to get it working fast but I would definitely add a task to tackle this at some point later. Do lmk if so

Reply:
ok, yeah, I was going to suggest that because I wanted to get the release trains going and this should be the last piece of the puzzle (I think). So maybe I change this to return an object for now and create a task to split in 2 actions. Is that cool?

Reply:
After thinking about it, I think it's better to leave as is, since returning an object would mean we would have to expose it from the plugin to the sdks, and not sure if that's a good idea. So I think we can merge this as is and just change to 2 actions later

Reply:
Created a task https://revenuecats.atlassian.net/browse/CSDK-417